commands/pull.js

```diff
@@ -30,6 +31,20 @@ function exec (cmd, opts = {}) {
   }
 }
 
+function spawn (cmd) {
+  const {spawn} = require('child_process')
+  return new Promise((resolve, reject) => {
+    let result = ''
+    let psql = spawn(cmd, [], {encoding: 'utf8', stdio: [ 'ignore', 'pipe', 'inherit' ], shell: true})
+    psql.stdout.on('data', function (data) {
+      result += data.toString()
+    })
+    psql.on('close', function (code) {
```

Comment:
oh that's maybe ok. Not having matching extensions seems like more of a warning scenario than an error one. Especially since it errors on non-equal extensions, but if there are more extensions on the target that should be fine.